src/test/kotlin/com/terraformation/backend/nursery/NurserySearchTest.kt

```diff
@@ -243,11 +252,11 @@ internal class NurserySearchTest : DatabaseTest(), RunsAsUser {
                           listOf(
                               mapOf(
                                   "species_id" to "$speciesId1",
-                                  "species_scientificName" to "Species $speciesId1",
+                                  "species_scientificName" to "Species 1",
```

Comment:
I'm a bit surprised by this change, shouldn't this also be dynamic since we're not explicitly passing an ID into the `insertSpecies` function? Or perhaps we should explicitly pass in a scientific name? 

Obviously the tests are still passing, so there's gotta be something I am misunderstanding.

Reply:
The numeric suffix here (the default value in `insertSpecies()`) is `"Species ${nextSpeciesNumber++}"` rather than using the ID; the number gets reset to 1 for each test. Species names aren't required to be unique across organizations so there's no issue with name collisions when tests run in parallel.

Reply:
I see, thanks for the clarification.